package-templates/react/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
```

Comment:
if we placed .eslintrc.js/babel/mocha at the root wouldn't it work without package configs?

Reply:
Yes it works, I removed the 'extend' part.

But still, we need those custom rules for no-extraneous-dependencies until there is a better solution for that problem. 
https://github.com/benmosher/eslint-plugin-import/issues/1174

Reply:
there seem to be several options within the thread you posted, none work? or maybe disable package checking altogether for now?

Reply:
- tried eslint-import-resolver-lerna with little success
- Didn't want to do the crazy stuff this guy did https://github.com/benmosher/eslint-plugin-import/issues/1174#issuecomment-509965883 Would rather have subpackage specific eslintrc.js for now